### DISCUSSION TITLE
What can I do for LinaEngine?

### DISCUSSION THREAD

**Discussion by C-BJ** [inanevin/LinaEngine#68] [Q&A] [answered]
I have thought of several points:
1.Reduce the use of existing code macro.
2.Update dependencies.
3.Do something bureaucratic.

**Comment by inanevin** [comment, marked as answer]
1- This is really but really ambigious. There is nothing like "reducing macros" without any explicit reasoning or bug fixes. Macros will be there, as they do serve a very specific purpose useful to many programming scenarios. Please open up a discussion about why you feel that certain macros should be removed, by reasoning how they will affect the architecture/engineering of the project software wise.

2- Currently dependencies are managed by LinaDependencies project.

3- Below

I really see the effort you are putting to make this project better, but any changes you want to make regarding the bueracracy of things should rather be suggesions & ideas in the discussions session. Contributions should be well-limited to code improvements, bug fixes, new features and so on. Any contribution regarding Wiki, Readme, Github Documents, icons & project description, purpose, structure and alike will be closed.

I appreciate & thank you for your motivation, but as said, I have a certain idea/motivation/vision for the Lina Project, and I am all open about reasonable discussing these, but not changing high-level organizational content without any discussion just based on PRs.

**Comment by inanevin** [comment]
And please also do not spam the issue/PR/discussion sessions, try to collect your ideas :).
